src/Controller/DownloadController.php

```diff
@@ -124,7 +124,7 @@ public function httpAction(string $id, DataStore $dataStore, LogActionItemEventD
         $logActionItemEventDispatcher->dispatch(
             array(
                 'actionName' => 'File Download',
-                'subjectEntityName' => $em->getClassMetadata(get_class($dataset))->getName(),
+                'subjectEntityName' => 'Pelagos\Entity\Dataset',
```

Comment:
I think the class name should stay, and instead we change the data. Just my $0.02.

Reply:
Probably we should do it as an improvement after we deploy flex